client/data/timeline/types.ts

```diff
@@ -0,0 +1,145 @@
+/* eslint-disable camelcase */
```

Comment:
Haven't checked each field, but thoughts are the same as with types for charge, maybe worth checking stripe lib for this.

Reply:
The Timeline is entirely our own thing, so I think we have to define our own types here.

That said, your point still stands; we might be able to use some of Stripe's pre-defined types here, but I'm personally leaning to managing our own types.

Reply:
Agree here unless we proxy some stripe objects (haven't checked this). Did you check?

Reply:
Yep, I did check. We do proxy some objects, but mostly we restructure the data to better fit the timeline, sometimes extracting just specific parts of an object.

Mostly the Timeline data structure is our own.